confluentinc/openapi-diff
# Add x-max-page-items extension.

What
----
Add extension to detect breaking changes to `x-max-page-items`.


References
----------


Test&Review
------------
* Add (incompatible)
* Remove (incompatible)
* Increase (compatible)
* Decrease (compatible)



* Ensure breaking changes are not reported for major version changes, where they're allowed. This may already be handled by `prototool`.
* Lint section in `prototool.yaml` can be disregarded, as we're not using that functionality yet.
* `prototool.yaml` can probably be trimmed.
